### DISCUSSION TITLE
10/24 박정혜 블로그 생성

### DISCUSSION THREAD

**Discussion by ghost** [Orm-camp/blog-feedback#25] [동글동글 3기 피드백]
<a href="https://yellowfiber.github.io/">https://yellowfiber.github.io/</a>

저는 깃허브 페이지(hydejack 테마)로 블로그를 만들었습니다. 
공부나 과제를 하면서 생긴 오류사항을 해결하는 과정을 위주로 작성할려고 합니다.
매주 하나라도.. 글 올리고 싶네요.

**Comment by ghost** [comment]
지윤환: 블로그 생성하시느라 고생 많으셨습니다. 
앞으로도 화이팅입니다!

**Comment by hyebinkimsdf** [comment]
정말 예쁜게 만드셨네요! 저도 열심히 만들어보려고합니다. 화이팅!

**Comment by mh0223** [comment]
블로그가 아기자기한 것 같아요!

**Comment by kimjigyun** [comment]
블로그 너무 예쁘게 만드셨네요 !!

**Comment by baaanjy** [comment]
블로그가 너무 예뻐요!!
